testsuite/MDAnalysisTests/topology/test_top.py

```diff
@@ -201,6 +210,76 @@ class TestPRMParser(TOPBase):
     expected_elems = None
 
 
+class TestPRMmultiParser(TOPBase):
```

Comment:
As above, multi is a bit hard to know what it does, so maybe add an issue / PR number and a comment on what it's checking?

Reply:
Done.